controllers/terminal/config/samples/terminal_v1_terminal.yaml

```diff
@@ -3,4 +3,9 @@ kind: Terminal
 metadata:
   name: terminal-sample
 spec:
-  # TODO(user): Add fields here
+  user: ccl
+  token: abc
+  apiServer: https://192.168.49.2:8443
```

Comment:
If this parameter is not filled in, will it be given a default value directly?Default apiserver address of the current cluster

Reply:
namespace default  is "default" namespace

Reply:
Yes, the default namespace is "default" if you not specify.

Reply:
I think `default` namespace is fine at this time. But support user's namespace will be better (may be next version) as cloud sealos version will give namespace and user info in.

Reply:
> I think `default` namespace is fine at this time. But support user's namespace will be better (may be next version) as cloud sealos version will give namespace and user info in.

if namespace param is empty , default namespace is default/

Reply:
Using apiserver.svc.cluster.local by default.